src/lib/components/Content.svelte

```diff
@@ -0,0 +1,85 @@
+<script lang="ts">
+  import { layoutBottomOffset } from "$lib/stores/layout.store";
+  import Toolbar from "$lib/components/Toolbar.svelte";
+  import MenuButton from "$lib/components/MenuButton.svelte";
+  import Back from "$lib/components/Back.svelte";
+
+  export let back = false;
+
+  // Enhance UI contrast for readability
+  export let contrast = false;
+</script>
+
+<div
+  class="content"
+  style={`--layout-bottom-offset: ${$layoutBottomOffset}px`}
```

Comment:
Why is this variable set inline?

Reply:
Because the CSS style needs a dynamic binded value calculated with JavaScript. Not sure I get your question, how would you solve this differently?

Note: this is not new but refactored.

Reply:
I don't know, but this is the 8th way we have to change style.

https://www.notion.so/Reusing-styles-06430b86e1f14e698dd7b44a99d2b6c2

Maybe this is the only way to fix it.

I also think we should push back designing some things if they add a big layer of complexity.

Reply:
Haha I still don't get your point here 😅. Not sure it is worth investing that deeper here, if you want to discuss it in a chat ping me anytime.

Reply:
No point for this PR, no worries. I am just not a fan of having so many ways of changing styles, and we added one more. Maybe it was the only way here. Just a comment to bring that topic, but I don't think you and @mstrasinskis are worried about that, so that's good.

Reply:
We have developed this particular piece of code together with @mstrasinskis 😜